src/libstd/sys/windows/stack_overflow.rs

```diff
@@ -1,11 +1,14 @@
 #![cfg_attr(test, allow(dead_code))]
 
+#[cfg(not(target_os = "uwp"))]
```

Comment:
This file might be a good example of just having a brand new file like `stack_overflow_uwp.rs` and use that instead of this one on uwp